docs/configure

```diff
@@ -492,7 +487,6 @@ dnslookup:
   domains = ! +local_domains
   transport = remote_smtp
   ignore_target_hosts = 0.0.0.0 : 127.0.0.0/8
-  headers_remove = X-Spam-Report:X-Spam-Score
```

Comment:
I haven't setup spamassassin on my latest FreeBSD-test system yet. I'd 
like to test this before I use it on a productive system.

On 2016-08-20 18:07, Rimas Kudelis wrote:
> In docs/configure [1]:
> 
>> @@ -492,7 +487,6 @@ dnslookup:
>>    domains = ! +local_domains
>>    transport = remote_smtp
>>    ignore_target_hosts = 0.0.0.0 : 127.0.0.0/8
>> -  headers_remove = X-Spam-Report:X-Spam-Score
> 
> I've amended the commit
> 
> --
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub [2], or mute the
> thread [3].
> 
> 	*
> 
> Links:
> ------
> [1] https://github.com/vexim/vexim2/pull/203#discussion_r75580896
> [2]
> https://github.com/vexim/vexim2/pull/203/files/bde6bf3db96db591e59704833a83b6462e53274e#r75580896
> [3]
> https://github.com/notifications/unsubscribe-auth/AGCFFDn6jDmNagsZQ0GX4r-UNMDylhGeks5qhyZMgaJpZM4JoLX7